releases/unreleased/add-support-for-outputting-main-dependencies.yml

```diff
@@ -0,0 +1,10 @@
+---
+title: Add support for outputting main dependencies
+category: added
+author: Lewi Uberg <lewi@uberg.me>
+issue: null
+notes: >
+  Add possibility to output only non-dev dependencies to a txt file with the
+  `-n, --non-dev` flag, and remove the default filename, forcing the user to
```

Comment:
We removed the `-n`. :)

Reply:
oh! sorry! it did not show in vscode when I searched the whole project for `-n`.